cpg-language-python/src/main/kotlin/de/fraunhofer/aisec/cpg/frontends/python/ExpressionHandler.kt

```diff
@@ -369,6 +377,11 @@ class ExpressionHandler(frontend: PythonLanguageFrontend) :
         return ret
     }
 
+    private fun isImport(name: Name): Boolean {
```

Comment:
I'm not very convinced this is a good approach. Am I understanding this correct, that a symbol `foo`, which happens to be an `ImportDeclaration` in one translation unit, would return `true` for another translation unit where this function is called? `foo` could be anything in another tu.

Reply:
Each translation unit is wrapped in its own module in our implementation, so if I resolve `foo` within a.py (which holds module `a`), it will not resolve `foo` of b.py, so I think we should be safe here.

A possibly better approach would be to get rid of the distinction between a call and a member call, but this this probably needs further discussion.